src/seedu/addressbook/data/person/Person.java

```diff
@@ -59,6 +65,8 @@ public UniqueTagList getTags() {
         return new UniqueTagList(tags);
     }
 
+    public int getSequenceNumber() { return sequenceNumber; }
```

Comment:
[Coding standard](https://oss-generic.github.io/process/codingStandards/CodingStandard-Java.html) violation! One more more of these problems detected:
1. brace style problem.